src/utils/wallet.js

```diff
@@ -229,7 +229,7 @@ const wallet = {
     const unknownTokens = [];
 
     // Iterating tokens to filter unregistered ones
-    for (const tokenUid of allTokens) {
+    for (const tokenUid of Object.values(allTokens)) {
```

Comment:
We should use the `Object.keys`

Reply:
Done on https://github.com/HathorNetwork/hathor-wallet/pull/587/commits/3ee5ac5cfba3a1e7dbe5387109159d06d4a34485